### ISSUE TITLE
Support mini_magick 5.0.0

### ISSUE THREAD

**Issue by jmaino** [janko/image_processing#125]
the gemspec currently restricts mini_magick to (>= 4.9.5, < 5), which prevents the update to the newly released 5.0.0 .

**Comment by janko** [comment]
Yes, I plan on doing that 👍🏻 That version constraint was intentional, as there will be changes needed to support MiniMagick 5.

**Comment by bvogel** [comment]
Any news here? Love your work.

**Comment by janko** [comment]
What specific functionality are you eager to get from MiniMagick 5.0? I'm not in a rush because that release was mostly about removing deprecated stuff.

**Comment by bvogel** [comment]
No rush, just a policy that we should avoid any library we use to fall two major releases behind, and as some are quite slow I got in the habit to keep up upon things. But taking this is on your roadmap, I'm fine waiting...

**Comment by Hessah95** [comment]
Hello... will this update be released soon?

**Comment by bvogel** [comment]
@Hessah95 you may have misunderstood this. This is only an issue discussing the possibility to even start an update to `mini_magick` v5. There is currently no code supporting this, you are free to open a PR to add support for this upgrade.

**Comment by jorihardman-aumni** [comment]
Alpine linux has [updated imagemagick to >v7](https://pkgs.alpinelinux.org/package/edge/community/s390x/imagemagick) which fully deprecates functionality used by mini_magick <5. In order to support our production docker images which run on alpine, we'd like to upgrade mini_magick to 5.X but cannot due to image_processing version restrictions. I'm not saying that this should drive prioritization of this issue, but I wanted to provide some context that will be shared with anyone using alpine docker images for deploying ruby apps.

**Comment by janko** [comment]
Are you referring only to the `convert` => `magick` deprecation? That shouldn't issue deprecation warnings on MiniMagick < 5, it should silence them.

But yeah, that's additional reason for me to get moving with the MiniMagick 5.x support 👍🏻

**Comment by lukeasrodgers** [comment]
I have a PR here https://github.com/janko/image_processing/pull/132 that begins this work. Happy to get feedback.

**Comment by janko** [comment]
https://github.com/janko/image_processing/pull/132 has been merged and released, so closing this one.

**Comment by jmaino** [comment]
thank you!
